Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -207,6 +217,25 @@ public float ScaleMinimum
         [Tooltip("Radius of the sphere collidable used in rotation handles")]
         private float rotationHandleDiameter = 0.035f;
 
+        [SerializeField]
+        [Tooltip("Only used if rotationHandlePrefab is specified. Determines the type of collider that will surround the rotation handle prefab.")]
+        private RotationHandlePrefabCollider rotationHandlePrefabColliderType = RotationHandlePrefabCollider.Sphere;
+        public RotationHandlePrefabCollider RotationHandlePrefabColliderType
+        {
+            get
+            {
+                return rotationHandlePrefabColliderType;
+            }
+            set
+            {
+                if (rotationHandlePrefabColliderType != value)
+                {
+                    rotationHandlePrefabColliderType = value;
+                    CreateRig();
```

Comment:
In a separate pr #4443  @djohnsomsft mentioed that CreateRig() has unintended consequences, at least in BoundingBox. For example, CreateRig() updates the min and max sizes in bounding box based on the object's current size.

Reply:
I'll hold off here until the dialog on that PR is resolved - I'd rather be consistent with how we're doing everything and I was using that PR as a reference point for consistency.

Reply:
Hey will, this is done now! Should be good to check in.